CONTRIBUTING.md

```diff
@@ -50,6 +50,13 @@ This will automatically update go.mod and go.sum.
 
 Commit your changes, and the next time `./package.sh` is run the `vendor/conjur-env` directory will be created with updated dependencies.
 
+When upgrading the version of Go for `conjur-env`, both the pre-built offline version and online version need to be
```

Comment:
This is kind of unfortunate. Is there any way we can tie these together? Like maybe have a `GO_VERSION` file that they both source from and automatically get the hash somehow? Not sure if you can parameterize a `Dockerfile` like that but it might be worth considering.

Reply:
I'm not sure, but it's a good idea. I've created an issue to look into iterating on this here: https://github.com/cyberark/cloudfoundry-conjur-buildpack/issues/41